internal/cli/atlas/clusters/clusters.go

```diff
@@ -57,7 +57,6 @@ func MongoCLIBuilder() *cobra.Command {
 		search.Builder(),
 		onlinearchive.Builder(),
 		connectionstring.Builder(),
-		availableregions.Builder(),
```

Comment:
pretty sure this is used for test, be sure no test breaks by making the command only available for atlas

Reply:
will keep an eye in waterfall results and only close the task afterwards